# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qemu/sasl2.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- qemu-0.13.0/ui/vnc-auth-sasl.h.vanilla	2010-12-14 16:37:21.663845652 +0000
+++ qemu-0.13.0/ui/vnc-auth-sasl.h	2010-12-14 16:37:26.498806373 +0000
@@ -27,7 +27,7 @@
 #define __QEMU_VNC_AUTH_SASL_H__
 
 
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 
 typedef struct VncStateSASL VncStateSASL;
 typedef struct VncDisplaySASL VncDisplaySASL;
